Source/WebKit/NetworkProcess/storage/CacheStorageManager.h

```diff
@@ -95,7 +91,7 @@ class CacheStorageManager : public CanMakeWeakPtr<CacheStorageManager> {
     std::pair<uint64_t, HashSet<WebCore::DOMCacheIdentifier>> m_pendingSize;
     String m_path;
     FileSystem::Salt m_salt;
-    CheckedRef<CacheStorageRegistry> m_registry;
+    Ref<CacheStorageRegistry> m_registry;
```

Comment:
I'm assuming you checked this doesn't introduce a ref-cycle.

Reply:
Yeah, `NetworkStorageManager` owns `CacheStorageRegistry` and a HashMap of `ClientOrigin` to `OriginStorageManager`, which in turns holds `CacheStorageManager.h` so I don't believe there is a circular ownership here.